.github/workflows/run_mmperf.yml

```diff
@@ -0,0 +1,92 @@
+name: mmperf
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
I'm not sure it makes sense to have a separate workflow that runs on pull_request. Why is this not part of the overall CI workflow?

Reply:
I added `pull_request` for testing it but it'll be removed once it's ready to be merged. Planning to run it nightly.

Reply:
Ah that makes more sense :-)

Reply:
Reminder to revert this before submitting. I suggest adding a "# DO NOT SUBMIT" comment. I've always meant to add a presubmit for that, but haven't gotten around it. But it at least stands out when you're scanning the code